Resources/Prototypes/Catalog/Fills/Books/bookshelf.yml

```diff
@@ -0,0 +1,77 @@
+- type: entity
+  id: BookshelfFilled
+  parent: Bookshelf
+  suffix: random filled
+  components:
+  - type: StorageFill
```

Comment:
make sure a bookshelf can contain all of these in the extremely rare case they all spawn (i think orGroup may be useful here too, more info in EntitySpawnEntry)

Reply:
For such rare cases, I have now increased the storage capacity to be able to store all these books.
I read, looked at the examples and tried orGroup, and then, as I understand it, it's a choice between subject A and subject B, and I couldn't understand why I should use it here, since I have nothing to choose from, and the appearance of all the books is accidental.